powermon/commands/response.py

```diff
@@ -0,0 +1,31 @@
+from powermon.dto.response_dto import ResponseDTO
```

Comment:
I've thought about that as well. At the moment the DTO is the only shared code the API needs to use which makes for a clean interface. I could be convinced otherwise but for now keeping them separate is clean.

Reply:
api could just import the DTO class out of the same file as the main class
having them together makes keeping them in sync easier was my thinking